Core/DISET/RequestHandler.py

```diff
@@ -239,24 +240,35 @@ def __doRPC(self, method):
     return self.__RPCCallFunction(method, args)
 
   def __RPCCallFunction(self, method, args):
+    """
+      Check arguments then call RPC function 
+      :type method: string
+      :param method: arguments sended by remote client
+      :return: S_OK/S_ERROR
+    """
     realMethod = "export_%s" % method
     gLogger.debug("RPC to %s" % realMethod)
     try:
+      # Get method we are trying to call
```

Comment:
De maniere generale, tu fais quelques erreurs d'anglais. je te le corrige par endroit, mais relis un peu partout, ce sont souvent les memes:
`Get THE method we are trying to call`